src/dvr/dvr_rec.c

```diff
@@ -356,6 +357,179 @@ dvr_sub_episode(const char *id, const char *fmt, const void *aux, char *tmp, siz
 }
 
 static const char *
+_dvr_sub_scraper_friendly(const char *id, const char *fmt, const void *aux, char *tmp, size_t tmplen, int with_genre_subdir)
+{
+  char date_buf[MAX(PATH_MAX, 512)] = { 0 };
+  char episode_buf[MAX(PATH_MAX, 512)] = { 0 };
+  const dvr_entry_t *de = aux;
+  /* Can't be const due to call to epg_episode_number_format */
+  /*const*/ epg_episode_t *episode = de->de_bcast ? de->de_bcast->episode : 0;
+
+  *tmp = 0;
+  const char *title    = lang_str_get(de->de_title, NULL);
+  const char *subtitle = lang_str_get(de->de_subtitle, NULL);
+  const char *desc     = lang_str_get(de->de_desc, NULL);
+
+  if (subtitle && desc && strcmp(subtitle, desc) == 0) {
+    /* Subtitle and description are identical so assume they are from
+     * bad OTA EIT.  Some OTA EIT often has a (long) summary which is
+     * put in to both subtitle and description. So we really don't
+     * want this to be used as the subtitle field.
+     */
+    subtitle = desc = NULL;
```

Comment:
I think that here should be only 'subtitle = NULL'.

Reply:
Agreed. The desc isn't used after that point but it makes sense to leave it non-null.
I'll make the change.